app.py

```diff
@@ -87,44 +87,66 @@ def experience(index = None):
         return jsonify({}), 201
     return jsonify({})
 
+
+def validate_education(body):
+    """ Return if education object is valid and if it's not what response and error code should be returned"""
+    required_fields = ['course', 'school', 'start_date', 'grade', 'logo']
+    if(body.get('grade') and not validate_grade(body.get('grade'))):
+        return False, jsonify({"error": "Invalid grade. The grade should be like A+ or F"}), 400
+    if body.get('start_date') and not validate_date_string(body.get('start_date')):
+        return False, jsonify({"error": "Invalid start date. Format should be `June 2023`"}), 400
+    if(body.get('end_date') and not validate_date_string(body.get('end_date'))):
+        return False, jsonify({"error": "Invalid end date. Format should be like `June 2023`"}), 400
+    if not validate_request(body, required_fields):
+        return False, jsonify({"error": "Invalid request. Required attributes are missing"}), 400    
+    return True, None, None
```

Comment:
can you put this with other validation functions in `utils.py`? that way we can shorten this file because it's getting pretty long.

also, do you need to `return True, None, None` or can you return something similar to other steps, i.e. `return True, jsonify({"status": "success"}), 200)`. **this is just a suggestion though! feel free to do it in your own style or double check how others are handling this**

Reply:
Moved validation function to utils.py.

"True, None, None" - Yes, I'm returning "None's" here because actual processing should happen later in web-method code, so I just put placeholder values, what is important is "True" which tells that validation is passed.